src/user/user.helpers.ts

```diff
@@ -0,0 +1,95 @@
+import HttpException from '../exceptions/HttpException';
+import UserNotFoundException from '../exceptions/UserNotFoundException';
+import { IHelperArgs } from '../interfaces/helpers.interface';
+import { IUser } from './user.interface';
+import { v4 as uuid_v4 } from 'uuid';
+import * as utils from '../utils';
+import InvalidIDException from '../exceptions/InvalidIDException';
+
+export const getUserByIdHelper = async (userHelpersArgs: IHelperArgs<IUser>): Promise<void> => {
+    const { db, request, response, next } = userHelpersArgs;
```

Comment:
Passing down the req/rest objects down to services considers to be a bad design/practice as it defeats the purpose of having controllers in the first place.
Consider moving any req/res operations to controllers.
Same logic applies to "next" callbacks.